pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -164,7 +164,8 @@ private void createOrUpdateRateLimiter(String tableNameWithType, ExternalView br
 
     if (brokerResource == null) {
       LOGGER.warn("Failed to init qps quota for table {}. No broker resource connected!", tableNameWithType);
-      buildEmptyOrResetRateLimiterInQueryQuotaEntity(tableNameWithType);
+      // It could be possible that brokerResourceEV due to ZK connection issue.
```

Comment:
Can you update the comment about `brokerResourceEV`? You meant `brokerResource is null`?